### PR TITLE
Add a user chip with a randomly chosen username and avatar.

### PR DESCRIPTION
**Pull request by aboodman**

### CODE REVIEW

File: backend/rds.ts [hunk 1]
```diff
@@ -90,17 +90,21 @@ export async function ensureDatabase() {
 }
 
 async function createDatabase() {
-  await executeStatementInDatabase(null, "CREATE DATABASE " + dbName);
+  await executeStatementInDatabase(
+    null,
+    `CREATE DATABASE ${dbName}
+    CHARACTER SET utf8mb4`
```
Comment (aboodman) [inline backend/rds.ts:96]: Needed for emoji

File: backend/rds.ts [hunk 2]
```diff
@@ -90,17 +90,21 @@ export async function ensureDatabase() {
...
+  );
 
   await executeStatement(`CREATE TABLE Cookie (
     Version BIGINT NOT NULL)`);
   await executeStatement(`CREATE TABLE Client (
-    Id VARCHAR(255) PRIMARY KEY NOT NULL,
+    Id VARCHAR(100) PRIMARY KEY NOT NULL,
```
Comment (aboodman) [inline backend/rds.ts:102]: The max size of keys is measured in bytes. By changing the character set, we exceeded it.

File: frontend/nav.module.css [hunk 3]
```diff
@@ -18,3 +18,20 @@
...
   opacity: 1;
 }
+
+.user {
+  color: white;
+  display: flex;
+  margin-left: auto;
+  margin-right: 12px;
+  margin-top: auto;
+  margin-bottom: auto;
```
Comment (arv) [inline frontend/nav.module.css:28]: margin: ...

File: pages/_document.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  static async getInitialProps(ctx) {
+    const initialProps = await Document.getInitialProps(ctx);
+    return { ...initialProps };
+  }
+
+  render() {
+    return (
+      <Html>
+        <Head>
+          <link rel="preconnect" href="https://fonts.gstatic.com" />
```
Comment (arv) [inline pages/_document.js:13]: does this really help? I would have thought link also causes prefetching?

**Review by arv — COMMENTED** [review]
LGTM

File: pages/_document.js (hunk 4 above)
Comment (aboodman) [reply pages/_document.js:13]: I'm not sure, this is what Google Font page suggested that I do.

**Pull request merged**